src/db/dbquery.cpp

```diff
@@ -407,39 +414,39 @@ std::string db::DBQuery::Query::to_json() const {
     json_res[query::endpoint::SRC_PORT] = port_src;
     json_res[query::endpoint::DST_PORT] = port_dst;
     json_res[query::endpoint::SFC_ID] = id_sfc;
-    (prt == protocol_type::TCP)? json_res[query::endpoint::PROTOCOL] = "tcp" :
+    (prt == protocol_type::TCP) ? json_res[query::endpoint::PROTOCOL] = "tcp" :
             json_res[query::endpoint::PROTOCOL] = "udp";
-    for (auto it = endpoints.cbegin(); it != endpoints.cend(); it++) {
-        if (it->get_endpoint_typology() == INGRESS_T) {
-            json_res[query::endpoint::SOCK_INGRESS] = it->get_socket_id();
-            json_res[query::endpoint::INGRESS_IP] = it->get_ip();
+    for (auto e : endpoints) {
```

Comment:
`e` could be `const`:
```suggestion
    for (const auto e : endpoints) {
```

Reply:
done